src/coreclr/src/pal/src/map/map.cpp

```diff
@@ -2137,6 +2137,18 @@ MAPRecordMapping(
     return palError;
 }
 
+static size_t OffsetWithinPage(off_t offset)
```

Comment:
This function implies it returns an offset, but its argument is "offset". Seems like the argument should be `addr` or `page`.

Reply:
There are two uses for this - in one it takes a VM address (destination) and in another it takes an offset to a file content. 
I am ok with renaming this `addr`